tests/integration/test_retry_producer.py

```diff
@@ -81,11 +91,17 @@ def test_retry_failed_payments(self):
         self.assertTrue(os.path.isfile(success_report))
 
         success_report_rows = CsvPaymentFileParser().parse(success_report, 10)
-        nb_success = len([row for row in success_report_rows if row.paid == PaymentStatus.PAID])
-        nb_hash_xxx_op_hash = len([row for row in success_report_rows if row.hash == 'xxx_op_hash'])
+        success_count = len([row for row in success_report_rows])
```

Comment:
@habanoz I updated the test to just check the row count without the condition `PAID` since the csv report is within the `done` folder meaning the rows mark successful transactions. To be sure I count the file number in the failed directory and assert it to zero. Let me know if the test is fine like this.